### ISSUE TITLE
Allow installing custom apps from custom source

### ISSUE THREAD

**Issue by MohammedNoureldin** [frappe/helm#174]
At the moment, to install apps, we can only pass the name of the app. However, sometimes it is also needed to install custom apps from custom source. This has to be also possible.

In addition, the custom apps must be persisted (at the moment there is no mount on their path, thus they just get lost when recreating the pod).

**Comment by MohammedNoureldin** [comment]
Well apparently we have a fundamental problem with installing/creating apps here.

I tried to create a very simple basic app and install it. Or course, the pod cannot be restarted because the container has no systemd, but I tried to recreate the whole pod (after adding a pvc and mount manually to persist the folder of the app insides apps). I was able to install the app, but when trying to call ERPNext in the browser, it does not work and I see in the logs that:

> ModuleNotFoundError: No module named '<app name>'

Something should be done in this regard. Am I missing something?
